contrib/debian/openzfs-zfsutils.install

```diff
@@ -0,0 +1,129 @@
+../tree/zfsutils-linux/* /
```

Comment:
Ah this is where it was being used. We should be able to get rid of it.

Reply:
Removed.